features-json/web-share.json

```diff
@@ -49,11 +49,11 @@
       "16":"n",
       "17":"n",
       "18":"n",
-      "79":"n",
-      "80":"n",
-      "81":"n",
-      "83":"n",
-      "84":"n"
+      "79":"y",
+      "80":"y",
```

Comment:
```suggestion
      "79":"n",
      "80":"n",
```

The feature was added in Edge 81:

![image](https://user-images.githubusercontent.com/20535393/90151868-7d41a900-dd87-11ea-8c78-fb130fd931bc.png)

![image](https://user-images.githubusercontent.com/20535393/90151885-82065d00-dd87-11ea-946d-829ac45619c6.png)